table/block_based/block_based_table_builder.cc

```diff
@@ -1511,7 +1487,7 @@ void BlockBasedTableBuilder::WritePropertiesBlock(
   if (ok()) {
     PropertyBlockBuilder property_block_builder;
     rep_->props.column_family_id = rep_->column_family_id;
-    rep_->props.column_family_name = rep_->column_family_name;
+    rep_->props.column_family_name = rep_->filter_context.column_family_name;
     rep_->props.filter_policy_name =
```

Comment:
Even if it might be redundant, it seems off that the column_family_id and column_family_name are not in the same struct/level.  I would prefer that the column_family_name stayed as a member of rep.

Even better is if it is rep->tboptions.column_family_name. :)